apps/desktop/src/lib/vbranches/types.ts

```diff
@@ -445,6 +445,12 @@ export class PatchSeries {
 	@Type(() => DetailedCommit)
 	upstreamPatches!: DetailedCommit[];
 
+	/**
+	 * A list of identifiers for the review unit at possible forges (eg. Pull Request).
+	 * The list is empty if there is no review units, eg. no Pull Request has been created.
+	 */
+	forgeIds!: ForgeIdentifier[];
```

Comment:
In the GitHub UI you can't create more than one PR for a branch. What is your reasoning here for an array of forge id's rather than just an optional one?